taxonomies/ingredients_processing.txt

```diff
@@ -303,8 +303,26 @@ de:gepökelt, gepökeltes
 
 de:ungepökelt
 
-# en:sweetened
+en:sweetened
 de:gesüßt, gesüßte
+es:endulzado, endulzada, endulzados, endulzadas
+nl:gezoet, gezoete
+pt:adoçado, adoçada
+
+en:unsweetened
+nl:ongezoet
+
+#<en:sweetened
+en:sugared
+de:gezuckert, gezuckerte
+fr:sucré, sucrée, sucrés, sucrées, sucree, sucrees #better not add fr:sucre and fr:sucres
```

Comment:
comments need to be on a separate line, otherwise it will break

Reply:
OK, repaired